### PR TITLE
Sentry bug bash

### PR DESCRIPTION
**Pull request by travisbloom-cb**
**Notes to reviewers**

**How has it been tested?**

- [ ] Locally
- [ ] Vercel Preview Branch
- [ ] Unit test
- [ ] Functional test

**Change management**
type=routine 
risk=low 
impact=sev5

### CODE REVIEW

File: src/utils/server/nft/mintPastActions.ts [hunk 1]
```diff
@@ -15,20 +15,23 @@ export async function mintPastActions(
...
       userId: userId,
-      actionType: { in: actionWithNFT },
       nftMintId: null,
     },
   })
 
   const analytics = getServerAnalytics({ userId: userId, localUser })
   for (const action of actions) {
+    const nftSlug = ACTION_NFT_SLUG[action.actionType]?.[action.campaignName]
+    if (!nftSlug) {
```
Comment (travisbloom-cb) [inline src/utils/server/nft/mintPastActions.ts:29]: https://stand-with-crypto.sentry.io/issues/5037009124/?environment=production&project=4506490717470720&query=is%3Aunresolved+timesSeen%3A%3E%3D10&referrer=issue-stream&statsPeriod=30d&stream_index=18

File: src/components/app/dtsiAvatar.tsx [hunk 2]
```diff
@@ -26,10 +28,15 @@ export const DTSIAvatar: React.FC<
       />
     )
   }
+  // https://stand-with-crypto.sentry.io/issues/5012094116/?environment=production&project=4506490717470720&query=is%3Aunresolved+timesSeen%3A%3E%3D10&referrer=issue-stream&statsPeriod=30d&stream_index=17
+  // this should never happen, adding additional debug info to figure out whats going on
+  if (!person.firstNickname && !person.firstName) {
```
Comment (travisbloom-cb) [inline src/components/app/dtsiAvatar.tsx:33]: https://stand-with-crypto.sentry.io/issues/5012094116/?environment=production&project=4506490717470720&query=is%3Aunresolved+timesSeen%3A%3E%3D10&referrer=issue-stream&statsPeriod=30d&stream_index=17

File: src/utils/web/safeStringify.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import * as Sentry from '@sentry/nextjs'
+
+// getting "JSON.stringify cannot serialize cyclic structures." errors.
+// this function should help identify the issues
+export function safeStringify(...params: Parameters<typeof JSON.stringify>) {
```
Comment (travisbloom-cb) [inline src/utils/web/safeStringify.ts:5]: https://stand-with-crypto.sentry.io/issues/5013863839/?environment=production&project=4506490717470720&query=is%3Aunresolved+timesSeen%3A%3E%3D10&referrer=issue-stream&statsPeriod=30d&stream_index=15

File: maybeDetectBrowser.test.ts [hunk 4]
```diff
@@ -19,13 +19,16 @@ const SAMPLE_UNSUPPORTED_BROWSERS = [
   'Mozilla/5.0 (Linux; Android 8.1.0; S23 Ultra Build/O11019; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/70.0.3538.110 Mobile Safari/537.36 Instagram 320.0.0.42.101 Android (27/8.1.0; 320dpi; 720x1544; alps; S23 Ultra; S23_Ultra; mt6735; es_MX; 570684018)',
   'Mozilla/5.0 (Linux; Android 8.1.0; LM-Q710(FGN) Build/OPM1.171019.019; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/70.0.3538.110 Mobile Safari/537.36 [FB_IAB/FB4A;FBAV/439.0.0.44.117;]',
   'Mozilla/5.0 (Linux; Android 10; SAMSUNG SM-A505FN) AppleWebKit/537.36 (KHTML, like Gecko) SamsungBrowser/12.1 Chrome/79.0.3945.136 Mobile Safari/537.36',
+  'Mozilla/5.0 (Android 9; Mobile; rv:122.0) Gecko/122.0 Firefox/122.0 DatadogSynthetics',
 ]
 
+function parseOption(str: string) {
+  const parsed = Bowser.parse(str)
+  // console.log(parsed)
```
Comment (lucasrmp-cb) [inline maybeDetectBrowser.test.ts:27]: 👀

**Review by lucasrmp-cb — APPROVED** [review]

**Pull request merged**